### DISCUSSION TITLE
Any debug log/info?

### DISCUSSION THREAD

**Discussion by darrenlmdse** [ericanastas/deploy-google-app-script-action#5] [Q&A] [answered]
Thanks for making the repo. I've followed the instruction but I haven't been able to make the auto push happen. Any idea where I can find some debug log/info of the auto deploy process?

**Comment by ericanastas** [comment, marked as answer]
Open the "Actions" tab:

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/using-workflow-run-logs

**Reply by darrenlmdse** [reply]
of course! The reason I couldn't find it before was simply because my deploy script was in the incorrect path. No wonder I couldn't find any log
